src/main/java/org/cactoos/text/IsEmpty.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017-2020 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.cactoos.text;
+
+import org.cactoos.Scalar;
+import org.cactoos.Text;
+
+/**
+ * Determines if text is empty or no.
+ * @since 0.47
+ */
+public final class IsEmpty implements Scalar<Boolean> {
```

Comment:
@kokodyn 
How is this class practical? Can we use it somewhere in the code base?

For example we could use `Func<Text, Boolean>` here https://github.com/llorllale/cactoos-matchers/blob/master/src/main/java/org/llorllale/cactoos/matchers/HasValuesMatching.java#L56. I guess technically you can pass it there as a method reference `IsEmpty::new`. 
But having `Scalar<Boolean>` in order to check is a bad idea imo, it creates short-lived objects.

Reply:
@kokodyn @andreoss let's keep it as a scalar for now, if short lived objects were a problem we would have to throw most of cactoos away :) (btw the JVM is very good at optimizing memory and execution for short lived objects)

Reply:
@andreoss @victornoel 
Practical using IsEmpty class to create matcher in org.llorllale.cactoos.matchers;
IsEmpty()
and then instead of using of new matcher:
        new Assertion<>(
            "The result text wasn't empty",
            new TextOf(input),
            new TextIs("")
        ).affirm();`
Like here:
https://github.com/yegor256/cactoos/blob/master/src/test/java/org/cactoos/io/HeadInputStreamTest.java#L79
We would use:
`        new Assertion<>(
            "The result text wasn't empty",
            new TextOf(input),
            new IsEmpty()
        ).affirm();`
I think it is much more readable with using IsEmpty